Update Russian docs for ver. 8.x

likhobory opened the pull request.

likhobory committed:
Update install-a-new-language.ru.adoc

likhobory committed:
Update content/8.x/admin/installation-guide/Upgrading.ru.adoc

likhobory committed:
Update System.ru.adoc (Repair ElasticSearch Index)

likhobory committed:
Update Workflow.ru.adoc

likhobory committed:
Update Upgrading.ru.adoc & add sectnums variables

likhobory committed:
Update Legacy-Migration.ru.adoc

likhobory committed:
Add Login-Throttling-Configuration.ru.adoc

likhobory committed:
Add LDAP-Configuration.ru.adoc

likhobory committed:
Add SAML-Configuration.ru.adoc

likhobory committed:
Update Surveys.ru.adoc

likhobory committed:
Update LDAP-Configuration.ru.adoc

    * Add Administration > Password Management menu LDAP configuration
    * Add Using Symfony Secrets

likhobory committed:
Update  SAML-Configuration.ru.adoc

* Update user auto creation description

likhobory committed:
Update Legacy-Migration.ru.adoc

likhobory committed:
Update Upgrading.ru.adoc

likhobory committed:
Update Downloading & Installing.ru.adoc (both versions)

likhobory committed:
Update Using_the_Upgrade_Wizard.ru.adoc & Upgrading.ru.adoc (both versions)

likhobory committed:
Add Microsoft-OAuth-Provider-HowTo.ru.adoc

likhobory committed:
Add InboundEmail-OAuth-HowTo.ru.adoc

likhobory committed:
Add OAuth-Provider-Overview.ru.adoc

likhobory committed:
Add InboundEmail-SecurityGroups.ru.adoc

likhobory committed:
Update content/admin/administration-panel/Emails/Email.ru.adoc

likhobory committed:
Update content/user/core-modules/Emails.ru.adoc

likhobory committed:
Update Compatibility Matrix.ru.adoc

likhobory committed:
Update Home Page.ru.adoc

likhobory committed:
Update Views.ru.adoc

likhobory committed:
Add Email-Compose-From-List.ru.adoc

likhobory committed:
Update content/8.x/admin/installation-guide/Legacy-Migration.ru.adoc

likhobory committed:
Update content/8.x/admin/installation-guide/Downloading & Installing.ru.adoc

likhobory committed:
Add 8.x/admin/installation-guide/Performance.ru.adoc

likhobory committed:
Remove trailing slash from the './bin/console/ clear:cache' command

likhobory committed:
Update existing content for ver. 8.3 (Russian)

likhobory committed:
Add some features description (Russian)

        content/8.x/features/_index.ru.adoc
        content/8.x/features/load-more.ru.adoc
        content/8.x/features/notifications.ru.adoc
        content/8.x/features/subpanel-filtering.ru.adoc
        static/images/ru/8.x/features/

likhobory committed:
Add Admin Page Documentation (Russian)

likhobory committed:
Update Compatibility Matrix.ru.adoc

likhobory committed:
Update System.ru.adoc

likhobory committed:
Update Administration-Panel.ru.adoc

likhobory committed:
Split  Downloading & Installing.ru.adoc

Add webserver-setup-guide.ru.adoc
Add running-the-cli-installer.ru.adoc
Add running-the-ui-installer.ru.adoc

likhobory committed:
Update some paths for :imagesdir:

likhobory committed:
Remove ../master/ from :imagesdir:

likhobory committed:
Remove :imagesdir: from files without images

likhobory committed:
Update content/8.x/admin/installation-guide/Upgrading.ru.adoc

likhobory committed:
Update README.ru.adoc

likhobory committed:
Update install-a-new-language.ru.adoc

likhobory committed:
Update Compatibility Matrix.ru.adoc

likhobory committed:
Update SAML-Configuration.ru.adoc

likhobory committed:
Update content/8.x/admin/installation-guide/Upgrading.ru.adoc

likhobory committed:
Update load-more.ru.adoc

likhobory committed:
Add Azure SAML Single Sign On Documentation (Russian)

yunusyerli1 committed:
Add field action documentation

likhobory committed:
Update Sales.ru.adoc

likhobory committed:
Add Trusted-Hosts-Configuration.ru.adoc

likhobory committed:
Update Compatibility Matrix.ru.adoc

likhobory committed:
Update Downloading & Installing.ru.adoc

likhobory committed:
Update webserver-setup-guide.ru.adoc

likhobory committed:
Update Search.ru.adoc and elasticsearch/Set up SuiteCRM.ru.adoc

likhobory committed:
Add quick-filters.ru.adoc

likhobory committed:
Update Legacy-Migration.ru.adoc

likhobory committed:
Update admin/installation-guide/_index.ru.adoc

likhobory committed:
Update elasticsearch/Introduction.ru.adoc

yunusyerli1 committed:
Fix asyncProcess

serhiisamko091184 committed:
Add Issue Voting page

likhobory committed:
Update Google Sync.ru.adoc

likhobory committed:
Update System.ru.adoc & Cases.ru.adoc

likhobory committed:
Update Legacy-Migration.ru.adoc

likhobory committed:
Update quick-filters.ru.adoc

likhobory committed:
Update Set up SuiteCRM.ru.adoc

likhobory committed:
Update Home Page.ru.adoc

likhobory committed:
Update Using the Upgrade Wizard.ru.adoc

likhobory committed:
Update Emails.ru.adoc

serhiisamko091184 committed:
Updated EN links

likhobory committed:
Fix links (ru)

likhobory committed:
Update load-more.ru.adoc

likhobory committed:
Update subpanel-filtering.ru.adoc

likhobory committed:
Update load-more.ru.adoc

likhobory committed:
Update formatting for 8.x

likhobory committed:
Update README.ru.adoc

likhobory committed:
Update Record Management.ru.adoc

jack7anderson7 committed:
Add Documentation for Pre Install Page

jack7anderson7 committed:
SuiteCRM 8.6 Release Notes

jack7anderson7 committed:
Update supported version for 8.6

jack7anderson7 committed:
Update Compatibility Matrix

clemente-raposo committed:
Replace ghcontributors api call with direct links

serhiisamko091184 committed:
Updated security-policy

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.6.0

serhiisamko091184 commented:
Hello @likhobory, 

thanks for contributing to SuiteCRM project, 

I've noticed there are conflicts present: 
![image](https://github.com/salesagility/SuiteDocs/assets/116086008/07da2cb4-1a8c-43f9-9018-a89d94690e65)

Would you be so kind to fix them? 

We'd like to update master branch, is it enough to pull these 2 commits to make branches synced (russian & master) from the content point of view? 
![image](https://github.com/salesagility/SuiteDocs/assets/116086008/15852fe9-5440-4c6c-b4f0-771c6529cf78)
    
Thanks in advance!

likhobory commented:
Hi @serhiisamko091184 
Yes, it will be enough.

serhiisamko091184 commented:
Hello @likhobory,

thanks for your quick reply, 

that's great, I'll pull them. 

When the conflicts are resolved, I'll be ready to test, approve, and get this PR merged.    

Thanks a lot! 

Regards,
Serhii

serhiisamko091184 commented:
Hey @likhobory,

allow me to draw your attention to this particular commit: 

https://github.com/salesagility/SuiteDocs/pull/646/commits/bf46be15398480e9ab04bbf8b031f0a441caada3

could you please update it to eliminate typos?

![image](https://github.com/salesagility/SuiteDocs/assets/116086008/2fde416a-ece5-467c-a2ec-c74a167a3941)

![image](https://github.com/salesagility/SuiteDocs/assets/116086008/0737eba6-0fce-478e-ad58-c9b345451922)

Many thanks! 

Regards,
Serhii

likhobory commented:
Hi @serhiisamko091184 
Thank you for paying attention, fixed.

jack7anderson7 committed:
Removed incorrect PR

likhobory committed:
Update running-the-ui-installer.ru.adoc

serhiisamko091184 committed:
Fix #610 - Issue about Page user/advanced-modules/PDFTemplates.adoc

serhiisamko091184 committed:
Fix #624 - Issue about Page 8.x/admin/installation-guide/running-the-cli-installer.adoc

jack7anderson7 committed:
Update Download link

jack7anderson7 committed:
Add links to assets

- install
- upgrade
- migration

serhiisamko091184 committed:
Updated upgrading v.8

jack7anderson7 committed:
Update ldap documentation

jack7anderson7 committed:
SuiteCRM 7.14.4 Release Notes

jack7anderson7 committed:
SuiteCRM 8.6.1 Release Notes

jack7anderson7 committed:
Update download links for 7 Release Notes

serhiisamko091184 commented:
Hello @likhobory,

Would you kindly give me an update on fixing conflicts, please? 

![image](https://github.com/salesagility/SuiteDocs/assets/116086008/5f9d7e9b-c489-44e5-a46e-ace733d9670c)

Thanks in advance!

likhobory commented:
Hi @serhiisamko091184
The contents of russian branch are not in sync with master branch, please force update russian branch.

kunanSA committed:
Update Customization.adoc

clemente-raposo committed:
Update upgrade page links on release notes

clemente-raposo committed:
Adjust the SuiteCRM 8 upgrade guide

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update 8.x/admin/installation-guide/running-the-cli-installer.ru.adoc

likhobory committed:
Update 8.x/admin/_index.ru.adoc

likhobory committed:
Update 8.x/admin/administration-panel/Administration-Panel.ru.adoc

likhobory committed:
Update 8.x/admin/configuration/LDAP-Configuration.ru.adoc

likhobory committed:
Update general-info.adoc

Invalid link to version 7 docs.

likhobory committed:
Update LDAP-Configuration.ru.adoc

likhobory committed:
Update Downloading & Installing.ru.adoc

likhobory committed:
Update content/8.x/admin/installation-guide/_index.ru.adoc

likhobory committed:
Update i18n/ru.toml

likhobory committed:
Split content/8.x/admin/installation-guide/Upgrading.ru.adoc

likhobory committed:
Fix links (ru)

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

serhiisamko091184 committed:
Changed the release date format

jack7anderson7 committed:
SuiteCRM 7.14.5 Release Notes

jack7anderson7 committed:
SuiteCRM 8.6.2 Release Notes

serhiisamko091184 committed:
Updated database-schema.adoc

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.6.2

clemente-raposo committed:
Update LDAP documentation with new changes from 8.7.0

clemente-raposo committed:
Update LDAP documentation with new changes from 8.7.0

clemente-raposo committed:
Add compatibility matrix for 8.7.0

clemente-raposo committed:
Add compatibility matrix for 8.7.0

clemente-raposo committed:
SuiteCRM 8.7.0 Release Notes

clemente-raposo committed:
SuiteCRM 8.7.0-beta Release Notes

clemente-raposo committed:
Update SAML documentation with changes from 8.7.0

clemente-raposo committed:
Update SAML documentation with changes from 8.7.0

clemente-raposo committed:
Update 8.7.0-beta Release Notes

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Merge branch 'pre-release' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.7-beta

clemente-raposo committed:
Update CVE

clemente-raposo committed:
Add 7.14 to 8.7.0+ migration documentation

mattlorimer committed:
Fix V8 API relationship reference

clemente-raposo committed:
Update field action documentation

clemente-raposo committed:
Update legacy migration documentation

clemente-raposo committed:
Update 8.7.0 release notes

likhobory committed:
Update content/8.x/admin/releases/8.7/_index.en.adoc

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.7.0

jack7anderson7 committed:
SuiteCRM 7.14.6 Release Notes

jack7anderson7 committed:
SuiteCRM 8.7.1 Release Notes

jack7anderson7 committed:
Update folder structure for features

jack7anderson7 committed:
Update folder structure for features

jack7anderson7 committed:
Update links for changed files

jack7anderson7 committed:
Update links for changed files

jack7anderson7 committed:
Add Two Factor Documentation

jack7anderson7 committed:
Add Two Factor Documentation

jack7anderson7 committed:
SuiteCRM 8.8.0-beta Release Notes

jack7anderson7 committed:
SuiteCRM 8.8.0-beta Release Notes

jack7anderson7 committed:
Add 8.8 compatibility matrix

jack7anderson7 committed:
Add 8.8 compatibility matrix

jack7anderson7 committed:
Update extension documentation for 8.8

jack7anderson7 committed:
Update extension documentation for 8.8

clemente-raposo committed:
Update 8.8 release notes

clemente-raposo committed:
Update 8.8 release notes

jack7anderson7 committed:
Update record mappers and save handler docs

jack7anderson7 committed:
Update record mappers and save handler docs

clemente-raposo committed:
Update 8.8 release notes

clemente-raposo committed:
Update 8.8 release notes

clemente-raposo committed:
Update 8.8 release notes

clemente-raposo committed:
Update 8.8 release notes

clemente-raposo committed:
Update 8.8 release notes

clemente-raposo committed:
Update timeout limit

clemente-raposo committed:
Update timeout limit

likhobory committed:
Update static/images/ru/8.x/admin/Installing/preinstall-error.png

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Merge branch 'pre-release' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.8.0-beta

jack7anderson7 committed:
Update Two Factor images and docs

jack7anderson7 committed:
Update 8.8 Release Notes

jack7anderson7 committed:
Update SuiteCRM 8.8 Release Notes

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.8.0

clemente-raposo committed:
Add steps on how to migrate extensions

mattlorimer committed:
Update issue voting to include PRs

jack7anderson7 committed:
Add SuiteCRM 8.9.0 Release Notes

jack7anderson7 committed:
Add new Campaigns Documentation

jack7anderson7 committed:
Add new Campaigns Documentation

jack7anderson7 committed:
Update Campaign Documentation

JanSiero committed:
Update Forking.adoc / change references to SalesAgility to SuiteCRM

change references to SalesAgility to SuiteCRM

jack7anderson7 committed:
Update Email Documentation

jack7anderson7 committed:
SuiteCRM 7.14.7 Release Notes

jack7anderson7 committed:
SuiteCRM 8.8.1 Release Notes

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Email.ru.adoc

likhobory committed:
Update Downloading & Installing.ru.adoc

likhobory committed:
Update Email.ru.adoc

likhobory committed:
Update InboundEmail-OAuth-HowTo.ru.adoc

likhobory committed:
Update Email.ru.adoc

jack7anderson7 committed:
Fix Outbound Email Documentation Link

jack7anderson7 committed:
Update releases link in community section

jack7anderson7 committed:
Add Scheduler Documentation

jack7anderson7 committed:
Add Async Validation Docs

jack7anderson7 committed:
Fix type in process handler docs

jack7anderson7 committed:
Add Init Default Process Docs

clemente-raposo committed:
Add new File Storage Documentation

jack7anderson7 committed:
Add extra configuration to attachment field

jack7anderson7 committed:
Update documentation section on Release Notes

jack7anderson7 committed:
Fix incorrect images dir

jack7anderson7 committed:
Add new footnotes documentation

jack7anderson7 committed:
Update 8.9.0 Release Notes

clemente-raposo committed:
Update attachment field documentation

clemente-raposo committed:
Update attachment field documentation

clemente-raposo committed:
Re-title async validators page

clemente-raposo committed:
Re-title backend default value calculation documentation page

clemente-raposo committed:
Update 8.9 release notes

jack7anderson7 committed:
Update 8.9.0 Release Notes

- fix media cycle link

jack7anderson7 committed:
Update field names and links

jack7anderson7 committed:
Update SuiteCRM 8.9.0 Release Notes

jack7anderson7 committed:
Add VardefConfigMapper documentation

clemente-raposo committed:
Add filter option documentation

clemente-raposo committed:
Add multirelate field documentation

clemente-raposo committed:
Move metadata related documentation to new metadata folder

clemente-raposo committed:
Update campaign documentation

clemente-raposo committed:
Update 8.9 release notes

clemente-raposo committed:
Update scheduler documentation

clemente-raposo committed:
Update file storage documentation

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

jack7anderson7 committed:
Update Supported Versions

jack7anderson7 committed:
Update Compatibility Matrix

jack7anderson7 committed:
Update Async Validators docs

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.9.0

likhobory committed:
Update links in ver. 8.9.0

likhobory committed:
Update Downloading & Installing.ru.adoc

jack7anderson7 committed:
SuiteCRM 7.14.8 Release Notes

jack7anderson7 committed:
SuiteCRM 8.9.1 Release Notes

jack7anderson7 committed:
Update Scheduler documentation

jack7anderson7 committed:
Update attachment documentation

clemente-raposo committed:
Update 8.9.1 and 7.14.8 CVE numbers

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.9.1

clemente-raposo committed:
Add email calendar invite documentation

clemente-raposo committed:
Add automatic email import documentation

mpuyosa91 committed:
Update weight of Release documentation

Adjusted the weight of the Release page to modify its position in the documentation hierarchy.

clemente-raposo committed:
Add OAuth2 Authorization Code grant documentation

jack7anderson7 committed:
SuiteCRM 7.15.0 Release Notes

mpuyosa91 committed:
Add new Calendar Sync and CalDAV documentation

Introduced comprehensive guides for Calendar Sync and CalDAV configuration. These include detailed synchronization mechanics, troubleshooting steps, and provider-specific instructions to assist users and administrators in configuring and managing calendar integrations effectively.

jack7anderson7 committed:
SuiteCRM 8.9.2 Release Notes

jack7anderson7 committed:
Update Surveys Image

mpuyosa91 committed:
Add replacement notice to Google Sync legacy documentation

Inform users that the Google Calendar Sync feature was replaced by the new Calendar Sync framework in v7.15+. The warning banner explains:
- Feature no longer exists in v7.15+
- Automatic migration occurs on upgrade
- Documentation preserved for v7.11-7.14 users

mattlorimer committed:
Update compatibility matrix

mattlorimer committed:
Update supported versions

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

jack7anderson7 committed:
Update compatibility matrix

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Update Russian docs for ver. 7.15

jack7anderson7 committed:
Add Composing Emails Documentation

jack7anderson7 committed:
Add variable injection link on Campaigns

rebase

jack7anderson7 committed:
Add Attach Documents Documentation

jack7anderson7 committed:
Add managing drafts documentation

jack7anderson7 committed:
Add documents management docs

jack7anderson7 committed:
Add notes documentation

jack7anderson7 committed:
Add email maximise documentation

jack7anderson7 committed:
SuiteCRM 7.14.9 Release Notes

jack7anderson7 committed:
SuiteCRM 8.9.3 Release Notes

jack7anderson7 committed:
SuiteCRM 7.15.1 Release Notes

jack7anderson7 committed:
Add Image field documentation

jack7anderson7 committed:
Add file field module builder documentation

mattlorimer committed:
Update out of date references

clemente-raposo committed:
Add documentation for Async Tasks and Manual Migration Tasks

clemente-raposo committed:
Add async task architecture and developer docs

jack7anderson7 committed:
Add PDF Templates Documentation

jack7anderson7 committed:
Add PDF Template Images

clemente-raposo committed:
Refine documentation for Manual Migration Tasks and clarify implementation details

clemente-raposo committed:
Refine documentation for Manual Migration Tasks and clarify implementation details

mattlorimer committed:
Update security policy

mattlorimer committed:
Update supported versions

jack7anderson7 committed:
Update Supported Versions

jack7anderson7 committed:
Add SuiteCRM 8.10 Compatibility Matrix

jack7anderson7 committed:
SuiteCRM 8.10 Release Notes

jack7anderson7 committed:
Update Field Types tables

jack7anderson7 committed:
Add is-true and is-false operators

jack7anderson7 committed:
Update Managing Drafts Docs

jack7anderson7 committed:
Add paused status and new config options to campaigns

jack7anderson7 committed:
Update Documents Docs

jack7anderson7 committed:
Update campaign images

jack7anderson7 committed:
Update email images

jack7anderson7 committed:
Update draft images

jack7anderson7 committed:
Add previous value operators

clemente-raposo committed:
Fix 8.10 release notes documentation toc

clemente-raposo committed:
Update 8.10 release date

clemente-raposo committed:
Update 8.10 release notes image field example

clemente-raposo committed:
Update 8.10 release notes version command description

clemente-raposo committed:
Add reference to Messenger Setup on Downloading & Installing guide

clemente-raposo committed:
Fix minimum version on async tasks documentation pages

clemente-raposo committed:
Update references to web user in Messenger Setup page

clemente-raposo committed:
Update reference to messenger setup in 8.10 release notes

mattlorimer committed:
Update 8.10 release notes

clemente-raposo committed:
Update symfony messenger setup docs

clemente-raposo committed:
Update links in 8.10 release notes

mattlorimer committed:
8.10 release notes

clemente-raposo committed:
Update maximised compose modal screenshot

clemente-raposo committed:
Update logic operators page

clemente-raposo committed:
Update async tasks admin section

clemente-raposo committed:
Update links on 8.10 release notes page

clemente-raposo committed:
Update links on async tasks developer section

clemente-raposo committed:
Update notes edit screenshot

likhobory committed:
Merge branch '7.15'

clemente-raposo committed:
Rename upgrading-82x-versions page to running-the-upgrade

jack7anderson7 committed:
SuiteCRM 8.10.1 Release Notes

likhobory committed:
Merge branch 'master' of https://github.com/salesagility/SuiteDocs

likhobory committed:
Merge branch 'master' of https://github.com/likhobory/SuiteDocs

likhobory committed:
Update Russian docs for ver. 8.10.1
